src/components/shards.html

```diff
@@ -8,22 +8,22 @@ <h4 class="modal-title"></h4>
                      src="assets/images/questionmark.png">
 
             </div>
-            <div class="modal-body" id="shardBody" data-bind='foreach: player._shardUpgrades'>
+            <div class="modal-body" id="shardBody" data-bind='foreach: Array(18)'>
```

Comment:
Should use the length of the shardEnum, maybe from a `ShardController.getDisplayableShards()`